src/Microsoft.ServiceHub.Framework/ServiceJsonRpcDescriptor.cs

```diff
@@ -279,6 +279,12 @@ public override RpcConnection ConstructRpcConnection(IDuplexPipe pipe)
 		return this.CreateConnection(jsonRpc);
 	}
 
+	/// <summary>
+	/// Initializes a new instance of <see cref="IJsonRpcMessageFormatter"/> for use in a new server or client.
+	/// </summary>
+	/// <returns>The new message formatter.</returns>
+	public IJsonRpcMessageFormatter GetFormatter() => this.CreateFormatter();
```

Comment:
Considering that the class already has a property named Formatter, The new method could be CreateMessageFormatter()?

Reply:
I like `CreateMessageFormatter`, any other opinions on the matter?